openhtf/core/test_state.py

```diff
@@ -365,7 +366,7 @@ def attach_from_file(self, filename, name=None, mimetype=None):
     """
     with open(filename, 'rb') as f:  # pylint: disable=invalid-name
       self.attach(
-          name if name is not None else filename, f.read(),
+          os.path.basename(name) if name is not None else filename, f.read(),
```

Comment:
I think you meant to do os.path.basename(filename)

name should be used as-is, since it's a more explicit way to provide the attachment name